src/accelerate/utils/versions.py

```diff
@@ -23,3 +26,34 @@
     import importlib.metadata as importlib_metadata
 
 torch_version = parse(importlib_metadata.version("torch"))
+
+
+def compare_versions(library_or_version: Union[str, Version], operation: str, requirement_version: str):
+    """Compares `library_or_version` to `requirement_version` with an `operation`
```

Comment:
```suggestion
    """
    Compares a library version to some requirement using a given operation.
```
I prefer to have complete English sentence describe the function :-)